test/dialects/insert-tests.js

```diff
@@ -402,8 +402,8 @@ Harness.test({
     string: 'INSERT INTO `post` (`userId`) SELECT `post`.`userId` FROM `user` INNER JOIN `post` ON (`user`.`id` = `post`.`userId`) WHERE (`post`.`tags` LIKE \'A%\')'
   },
   mssql: {
-    text  : 'INSERT INTO [post] ([userId]) SELECT [id] FROM [user] WHERE ([name] LIKE @1)',
-    string: 'INSERT INTO [post] ([userId]) SELECT [id] FROM [user] WHERE ([name] LIKE \'A%\')'
+    text  : 'INSERT INTO [post] ([userId]) SELECT [post].[userId] FROM [user] WHERE ([user].[name] LIKE @1)',
+    string: 'INSERT INTO [post] ([userId]) SELECT [post].[userId] FROM [user] WHERE ([user].[name] LIKE \'A%\')'
```

Comment:
This test is still failing - I'm not sure what the proper syntax should be in this case. :(